### DISCUSSION TITLE
How do I setup Doxygen locally for testing the latest changes?

### DISCUSSION THREAD

**Discussion by cloud-sonic** [doxygen/doxygen#11257] [General]
I'm unable to find a way to test the current changes on my device(Windows), except the releases.

**Comment by albert-github** [comment]
The latest changes can be found as artifacts under Actions under Github (see https://github.com/doxygen/doxygen/actions) and go to the latest succeeded (non Coverity) master build.  At the bottom of the page you will find the Artifacts with just the executables (and the latest documentation).

**Comment by paule32** [comment]
Hello cloud-sonic,
I initialled start a project for my own.
But I think, it can be grow, and useful by other developers.
So, I make it public to anyone.
It is not ready, yet.
But I worked on at my spare free time.

You can take a look to this repro at:
[doxyhelp](https://github.com/paule32/HelpNDocTools)

I will do my best, to bring it on front.
But don't be tired.

Thanks for reading and watching
Blacky
